doc/source/data/working-with-llms.rst

```diff
@@ -50,10 +58,12 @@ It contains the model name, the number of GPUs to use, and the number of shards
             sampling_params=dict(
                 temperature=0.3,
                 max_tokens=250,
-            )
+            ),
+            **row
```

Comment:
I don't think you need this in preproess. All columns are automatically carried over until the postprocess.

Reply:
oh that's very unintuitive! OK I will document that.